src/background/background.js

```diff
@@ -1,12 +1,11 @@
 // eslint-disable-next-line import/extensions
-import getBrowserPolyfill from '../custom-polyfill.js';
+import { getBrowserPolyfill, apiUrl } from '../common.js';
 
-const apiUrl = 'https://bff.ecoindex.fr/api';
 let tabUrl = '';
 const currentBrowser = getBrowserPolyfill();
-
+const defaultColor = '#5b5b5b';
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**🎯 suggestion(non-blocking):** Peut être exporter dans le common.js ?